modules/virt-creating-bridge-nad-web.adoc

```diff
@@ -10,6 +10,12 @@
 
 Network administrators can create network attachment definitions to provide layer-2 networking to pods and virtual machines.
 
+The `NetworkAttachmentDefinition` is a custom resource that exposes layer-2 devices
```

Comment:
Lines 11 and 16-17 are duplicate content. Also, I just edited this module for another bug, so you probably need to rebase to get the latest changes in your PR.